app/presenters/vendor_api/base.rb

```diff
@@ -40,6 +40,12 @@ def active_version_in_retrieved_version?(version)
       minor_version_number(active_version) >= minor_version_number(version)
     end
 
+    def version_available_in_environment?(version)
+      return true unless HostingEnvironment.production?
```

Comment:
do we need to add logic here for sandbox?

Reply:
I added another spec but we don't need any extra logic as you can only access the presenters for sandbox for up to the fixed version, regardless of it being prerelease or not.